src/react-apps/applications/receipt/src/features/receipt/containers/Receipt.tsx

```diff
@@ -0,0 +1,204 @@
+  };
+
+  React.useEffect(() => {
+    if (!application && instance) {
+      fetchApplication();
+    }
+  }, [instance]);
+
+  React.useEffect(() => {
+    fetchInstance();
+    fetchParty();
+    fetchOrganizations();
+    fetchUser();
+    fetchLanguage();
+  }, []);
+
+  return (
+    <>
+      <AltinnAppHeader
+        logoColor={theme.altinnPalette.primary.blueDarker}
+        headerColor={theme.altinnPalette.primary.blue}
+        party={party ? party : {} as IParty}
+        // tslint:disable-next-line: max-line-length
+        userParty={user ? user.party : {} as IParty}
+      />
+        <AltinnModal
+          classes={props.classes}
+          isOpen={true}
+          onClose={handleModalClose}
+          hideBackdrop={true}
+          hideCloseIcon={isPrint}
+          headerText={getLanguageFromKey('receipt_platform.receipt', language)}
+        >
+        {isLoading() &&
+          <AltinnContentLoader/>
+        }
+        {!isLoading() &&
+          <AltinnReceipt
+            title={getTitle()}
+            body={getLanguageFromKey('receipt_platform.helper_text', language)}
+            collapsibleTitle={getLanguageFromKey('receipt_platform.attachments', language)}
+            attachments={getAttachments()}
+            instanceMetaDataObject={getInstanceMetaDataObject(instance, party, language, organizations)}
+            titleSubmitted={getLanguageFromKey('receipt_platform.sent_content', language)}
+          />
+        }
+        </AltinnModal>
+    </>
+  );
+}
+
+export const getInstanceMetaDataObject = (instance: IInstance, party: IParty,  language: any, organizations: any) => {
+  const obj = {} as any;
+  let dateSubmitted;
+  if (instance.data) {
+    const lastChanged = instance.data.filter((elem) => elem.elementType === 'default')[0].lastChangedDateTime;
+    dateSubmitted = moment(lastChanged).format('DD.MM.YYYY / HH:mm');
+  }
+  obj[getLanguageFromKey('receipt_platform.date_sent', language)] = dateSubmitted;
+  let sender: string = '';
+  if (party && party.person.ssn) {
+    sender = `${party.person.ssn}-${party.person.name}`;
+  } else if (party) {
+    sender = `${party.orgNumber}-${party.name}`;
+  }
+  obj[getLanguageFromKey('receipt_platform.sender', language)] = sender;
+  obj[getLanguageFromKey('receipt_platform.receiver', language)] = getOrganizationDisplayName(instance, organizations);
+  obj[getLanguageFromKey('receipt_platform.reference_number', language)] = getInstanceId();
+  return obj;
+};
+
+export const getOrganizationDisplayName = (instance: IInstance, organizations: any ): string => {
+  if (!organizations) {
+    return instance.org.toUpperCase();
+  } else {
+    // TODO: fetch this language based on language cookie
+    if (organizations.orgs[instance.org]) {
+      return organizations.orgs[instance.org].name.nb.toUpperCase();
+    } else {
+      return instance.org.toUpperCase();
+    }
+  }
+};
+
```

Comment:
Why are these two functions above not in utils as helper-functions?

Reply:
Fixed.

Reply:
Good point. Moved to utils/receipt.ts